config.toml

```diff
@@ -225,7 +225,7 @@ memo_prefix = ''
 
 # Specify custom ICS23 proof-specs for the chain (serialized as JSON).
 # Default: [`ProofSpecs::cosmos()`](modules/src/core/ics23_commitment/specs.rs)
-proof-specs = '''
+proof_specs = '''
```

Comment:
I think we should hide this option (i.e., delete all these lines from the config.toml). It seems this option is strictly for testing (at least at the moment), and it is a distraction towards relayer operators to keep it here.

Any thoughts @hu55a1n1, am I missing something wrt to the use-cases of `proof_specs`?

Reply:
I agree! I think `proof_specs` is only used for testing currently so we can remove it from here. 👍 We can add it back once `hermes` is able to support non-SDK chains.

Reply:
OK, will remove.